examples/DocQA/docker/llama_stack_start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Sleep for 45 seconds
+sleep 45
+# Print a message indicating the start of llama-stack docker
+echo "-----starting to llama-stack docker now---------"
+# Install required packages
+pip install gradio
+
+# Check if GPU is enabled and run ingest files script accordingly
+if [ "$USE_GPU" = true ]; then
+  echo "Using GPU to ingest files"
+  pip install docling
+  python /root/DocQA/scripts/ingest_files.py --input_dir /root/rag_data/
+fi
+# Print a message indicating the start of llama-stack server
+echo "starting the llama-stack server"
+# Run llama-stack server with specified config and disable ipv6
+python -m llama_stack.distribution.server.server --yaml-config /root/my-run.yaml --disable-ipv6&
```

Comment:
I think this should be re-architected a bit. Why isn't the docker running two services for this purpose:
- one for running the llama stack server
- one for running the RAG app? that entrypoint can just be `python /root/DocQA/app.py`

Reply:
I was hoping to use `LlamaStackDirectClient `, but I am not sure if `LlamaStackDirectClient` supports MemoryBank connection to ChromaDB.